tasks/managed/populate-release-notes/populate-release-notes.yaml

```
@@ -282,6 +282,22 @@ spec:
                 CVEsJson=$(jq --argjson cve "$cveJson" '.cves.fixed += $cve' <<< "$CVEsJson")
             done
         
+            created_label=$(jq -r '.metadata.labels[]? | select(.name == "org.opencontainers.image.created")
+                | .value' <<< "$component")
+
+            timestamp_tag=""
+            if [[ -n "$created_label" ]]; then
+                # Attempt to convert RFC3339 date to unix epoch (seconds)
+                if ! timestamp_tag=$(date -d "${created_label}" +%s 2>/dev/null); then
+                    echo "Warning: Component '${name}' has 'org.opencontainers.image.created' label" \
+                        "('$created_label') but it could not be parsed as a date." >&2
+                    timestamp_tag=""
+                fi
+            else
+                 echo "Warning: Component '${name}' is missing the 'org.opencontainers.image.created'" \
```

nit: one extra whitespace :-)

```suggestion
                echo "Warning: Component '${name}' is missing the 'org.opencontainers.image.created'" \
```

---

Fixed.